CSS/Components/Modal/modal.css

```diff
@@ -0,0 +1,31 @@
+.modal-backdrop {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: var(--modal-backdrop);
+  z-index: 1;
+}
+
+.display-none{
+  display: none;
```

Comment:
You can put this common Utility file so that every required component can use it.